src/routers/__tests__/routerTestHelper.js

```diff
@@ -0,0 +1,56 @@
+import NavigationActions from '../../NavigationActions';
+
+// A simple helper that makes it easier to write basic routing tests
+// We generally want to apply one action after the other and check router returns correct state
+// it's often convenient to manipulate a structure that keeps the router state to avoid
+// creating many state1, state2, state3 local variables which are prone to typos...
+
+const debug = false;
+
+const defaultInitAction = {
+  type: NavigationActions.INIT,
+};
+
+export const getRouterTestHelper = (router, initAction = defaultInitAction) => {
+  let state = router.getStateForAction(initAction);
+
+  const applyAction = action => {
+    debug && console.debug('\n');
```

Comment:
Can remove these debug logs if needed. Just find them convenient when I'm not sure what's happening. Probably need this because I'm not really familiar with a proper test debugging workflow.

Reply:
Actually just found this and it seems works fine in my IDE :D https://jestjs.io/docs/en/troubleshooting

Reply:
Let's remove these logs